package.json

```diff
@@ -12,19 +12,19 @@
     "test": "test"
   },
   "dependencies": {
-    "d3-dsv": "^0.1.9"
+    "d3-dsv": "^1.0.3"
```

Comment:
I don't know much about those libs. I just saw that there is a new version of d3-dsv and that it can be easily updated.